cinema/serializers.py

```diff
@@ -78,3 +98,58 @@
     class Meta:
         model = MovieSession
         fields = ("id", "show_time", "movie", "cinema_hall")
+
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+        representation["taken_places"] = [
+            {"row": ticket.row, "seat": ticket.seat}
+            for ticket in instance.tickets.all()
+        ]
+        return representation
```

Comment:
The `to_representation` method in `MovieSessionDetailSerializer` adds a custom field `taken_places` to the serialized output, which lists the row and seat numbers of all tickets for the session. This is a useful customization for detailed session views.